app/models/concerns/localizable.rb

```diff
@@ -0,0 +1,13 @@
+# Add to a model that can be localized independent of the locale but with the
+# same set of languages (e.g. News, TheftAlertPlan models).
+#
+# Expects a `language` column defined on the corresponding table.
+module Localizable
+  extend ActiveSupport::Concern
+
+  included do
+    enum language: I18n.available_locales.sort.map.with_index.to_h.freeze
```

Comment:
Beautiful! I love this concern.

Except - it makes accessing the actual value of the enum a little difficult - which might need to happen, especially in this version of Rails.

I _think_ putting this in a class method resolves it?

If you think this is a non-issue, or you have a different solution, let me know!

Reply:
> it makes accessing the actual value of the enum a little difficult

Do you mean the enum hash? The Rails inflector helps out with that, but let me know if you had something else in mind.

```ruby
TheftAlertPlan.languages
=> {
  "en" => 0,
  "nl" => 1
}
```

Reply:
OH SNAP. Yes, that, it's a non-issue 😊